velox/connectors/hive/storage_adapters/hdfs/HdfsReadFile.h

```diff
@@ -33,19 +37,28 @@ struct HdfsFile {
   void open(hdfsFS client, const std::string& path) {
     client_ = client;
     handle_ = hdfsOpenFile(client, path.data(), O_RDONLY, 0, 0, 0);
+#ifdef VELOX_ENABLE_HDFS3
     VELOX_CHECK_NOT_NULL(
         handle_,
         "Unable to open file {}. got error: {}",
         path,
         hdfsGetLastError());
```

Comment:
Can we just check `` in `hdfsGetLastError`, return  "" or "Can not get error message when VELOX_ENABLE_HDFS3" when #ifdef VELOX_ENABLE_HDFS3, then we don't need to modify so much code